### DISCUSSION TITLE
Why does default husky/_/.gitignore ignore everything?

### DISCUSSION THREAD

**Discussion by bestickley** [typicode/husky#1441] [Q&A] [answered]
Hi, I was surprised to see that the default husky/_/.gitignore ignores everything. See VS Code screenshot below. If all scripts are gitignored, then other devs who clone the repo won't have them, right? Is this intentional? Is the workflow for us to add !pre-commit or whatever script we want to keep after the *? Then we would also have to un gitignore h as well, right?
![image](https://github.com/typicode/husky/assets/35735118/b149cbdc-70f0-47db-82fa-fb591a6dd0b8)

**Comment by typicode** [comment, marked as answer]
They're generated files, so they're not versioned. They're created when `husky` commands run.
The suggested workflow is to run `husky` using `prepare`. But you're free to adapt it.

**Reply by bestickley** [reply]
@typicode, thanks for answer. Running `pnpm run prepare` created this file structure. I understand now, I thought I needed to edit the .husky/_/pre-commit file from the [docs](https://typicode.github.io/husky/how-to.html#manual-setup), but I actually need to edit the .husky/pre-commit file (notice no _). This was a little confusing to me. Hopefully this discussion helps others. Thanks, again!
